docs/templates/using/cli.md

```diff
@@ -46,6 +46,12 @@ Test whether a model is defined correctly and whether is execution using the exa
 kipoi test ~/.kipoi/models/Basset/example_files
 ```
 
+In detail, `kipoi test <model>` checks for three things:
```

Comment:
Maybe add ```--source=kipoi``` as well? :)

Reply:
is there a difference in the tests when one removes this flag?

Reply:
So if you write ```kipoi test``` you get the following
```
usage: kipoi test [-h] [--source {kipoi,github-permalink,dir}]
                  [--batch_size BATCH_SIZE] [--keep_metadata] [-o OUTPUT] [-s]
                  [-e EXPECT]
                  model
```
```source=dir``` will pick up changes in the model.yaml in your local directory but ```source=kipoi``` will continue to refer to the content kipoi! Does it make sense?

Reply:
Got it!